tools/wptrunner/wptrunner/browsers/firefox.py

```diff
@@ -119,13 +119,13 @@ def browser_kwargs(logger, test_type, run_info_data, config, **kwargs):
                       "e10s": kwargs["gecko_e10s"],
                       "enable_fission": run_info_data["fission"],
                       "stackfix_dir": kwargs["stackfix_dir"],
-                      "binary_args": kwargs["binary_args"],
+                      "binary_args": kwargs["binary_args"].copy(),
                       "timeout_multiplier": get_timeout_multiplier(test_type,
                                                                    run_info_data,
                                                                    **kwargs),
                       "leak_check": run_info_data["debug"] and (kwargs["leak_check"] is not False),
                       "asan": run_info_data.get("asan"),
-                      "chaos_mode_flags": kwargs["chaos_mode_flags"],
+                      "chaos_mode_flags": kwargs["chaos_mode_flags"].copy(),
                       "config": config,
```

Comment:
Do we need to copy the `config` as well or is that only a problem with `kwargs`? There are also other lines in this file which assign lists/dicts directly like `options["args"] = kwargs["binary_args"]`. What's with those?

Reply:
Actually those `chaos_mode_flags` are of type `int` and there is no need to actually use `copy` here. I'll push an update soon which removes this change, which will make the `--repeat-until-unexpected` argument work again.

Reply:
At least the same changes are also required for the `firefox_android.py` file, which I'm going to update as well.